qa/vagrant-helpers.rb

```diff
@@ -34,10 +50,17 @@ def self.parse(lines)
 
     def self.execute(cmd)
       Open3.popen3(cmd) do |stdin, stdout, stderr, wait_thr|
-        { :stdout => stdout.read.chomp, :stderr => stderr.read.chomp,
-          :exit_status => wait_thr.value.exitstatus }
+        ExecuteResponse.new(stdin, stdout.read.chomp, stderr.read.chomp, wait_thr.value.exitstatus)
```

Comment:
No it doesn't but this https://github.com/elastic/logstash/pull/5180/files does.